runtime/minor_gc.c

```diff
@@ -431,7 +431,7 @@ static void oldify_mopup (struct oldify_state* st, int do_ephemerons)
   if (redo) goto again;
 }
 
-void caml_empty_minor_heap_domain_clear(caml_domain_state* domain, void* unused)
```

Comment:
I looked at why we have this unused parameter here. It was introduced in 123b25d87fa5269a1e829de207fbe180bdb8f64c (at the it was necessary to conform to the type expectations of `caml_run_on_all_running_domains_during_stw` and it stopped being useful in 6800745595aad8fae649156b4a56b97550eba2a7.